### DISCUSSION TITLE
Do we have feature auto-instrumentation for metrics or plan for it?

### DISCUSSION THREAD

**Discussion by haingo-kenshjn** [open-telemetry/opentelemetry-java-instrumentation#3481] [Q&A]
Hi team,

The current auto-instrumentation for tracing look awesome. Do we have feature auto-instrumentation for metrics or plan for it?
Appreciate if you could kindly assist on this.

Thanks,
Hai Ngo,

**Comment by iNikem** [comment]
When Metric API and SDK start stabilising we certainly are going to consider what metric we can collect automatically. All ideas are welcomed!

**Reply by mateuszrzeszutek** [reply]
There's also this roadmap that was published a while ago: https://medium.com/opentelemetry/opentelemetry-metrics-roadmap-f4276fd070cf
